packages/devtools_app/lib/src/ui/flutter/flutter_icon_renderer.dart

```diff
@@ -8,6 +8,7 @@ library icon_renderer;
 import 'package:flutter/material.dart';
 import 'package:flutter/widgets.dart';
 
+import '../fake_flutter/_real_flutter.dart';
```

Comment:
Should this be a package:flutter/something.dart import?

Reply:
looks like it isn't needed at all. removed.